api/src/main/java/org/eclipse/microprofile/config/spi/Converter.java

```diff
@@ -67,7 +68,7 @@
     /**
      * Configure the string value to a specified type
      * @param value the string representation of a property value
-     * @return the converted value
+     * @return the converted value or null if value is null
```

Comment:
OK I would update the spec then to say all Builtin converters return null if a null value is provided. I still think an implementer of a ConfigSource would want some guidance on null handling.